src/frontend/typer/oppgave.ts

```diff
@@ -70,6 +70,7 @@ enum Dokumentstatus {
 }
 
 export interface IOppgave {
+    [key: string]: string;
```

Comment:
What is this used for?

Reply:
The fields need to be accessed dynamically by key. There will be typescript error if there is no such definition.